pubspec.yaml

```diff
@@ -6,51 +6,36 @@ authors:
   - Workiva UI Platform Chapter <uip@workiva.com>
   - Workiva App Frameworks Team <appframeworks@workiva.com>
 environment:
-  sdk: ">=1.24.2 <2.0.0"
+  sdk: '>=1.24.2 <=3.0.0'
 dependencies:
-  analyzer: ">=0.30.0+4 <=0.31.0"
-  barback: ">=0.15.2 <=0.15.2+14"
+  # analyzer >= 0.33.0 breaks the builder, and analyzer < 0.32.4 is Dart 2 incompatible. Wdesk currently resolves to
+  # 0.30.x, so we need the lower bound
+  analyzer: ">=0.30.0 <0.33.0"
```

Comment:
I understand why this lower bound is needed, but have we verified that the builder actually works with 0.30.0?

Reply:
We'll need to increase the upper bound to <0.34.1 in order for the dart 2 version of over_react to be compatible with semver-audit-dart.

Reply:
Nope, that's a good point. I'll test that out

Reply:
Raise this to be at least the latest 0.32.x version that it currently resolves to.